financeiro/2024-06.beancount

```diff
@@ -0,0 +1,213 @@
+;; -*- mode: beancount -*-
+;; extratos/2024-06-bb.ofx
+
+2024-06-03 * "" "PIX - REJEITADO - 03/06 12:49 CPF/CNPJ NAO PERTENCE AO T / DEP"
+  comprovante: ""
+  Assets:Bancos:BB  152.60 BRL
+  Income:APyB:Estorno
+
+2024-06-03 * "" "PIX - REJEITADO - 03/06 12:50 CPF/CNPJ NÃO PERTENCE AO T / DEP"
+  comprovante: ""
+  Assets:Bancos:BB  152.60 BRL
+  Income:APyB:Estorno
```

Comment:
Nos casos de estorno, não faria mais sentido voltar para a conta de onde o PIX saiu inicialmente? Também seria bom ter uma referencia para tal transação, né?

Reply:
Vou inserir a referência. Não entendi o que tu quis dizer com "voltar para a conta de onde o PIX saiu inicialmente". O Pix saiu do Banco do Brasil e voltou pra ele. Tu fala relacionado ao Income?

Reply:
> O Pix saiu do Banco do Brasil e voltou pra ele. Tu fala relacionado ao Income?

Isso, to falando especificamente sobre ter uma conta no _Income_ para estorno. 

Se um PIX saiu do BB e foi para uma conta de _expense_ X, o estorno tem que sair da conta X e voltar para o BB.

Reply:
adicionada referência e alterada conta